### PR TITLE
update url

### PR DESCRIPTION
**Pull request by jburel**
This PR updates the link and points to mybinder where the notebooks can now be run
cc @sbesson

### CODE REVIEW

File: _includes/head-idr.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
                             <li><a href="{{ "/api.html" | prepend: site.baseurl }}">API Access</a></li>
                             <li><a href="{{ "/download.html" | prepend: site.baseurl }}">Data download</a></li>
                             <li><a href="{{ "/itr.html" | prepend: site.baseurl }}">Image Tools Resource (ITR)</a></li>
-                            <li><a href="/jupyter">Virtual Analysis Environment (VAE)</a></li>
+                            <li><a href="https://mybinder.org/v2/gh/IDR/idr-notebooks/master">Virtual Analysis Environment (VAE)</a></li>
```
Comment (sbesson) [inline _includes/head-idr.html:47]: While updating the menu makes sense, we are still contemplating to redirecting these resources to the EBI Binder resources when these achieve production status.

Another solution would be to continue using an intermediate URL and adjust the redirects under the hood as we migrate?

**Review by sbesson — COMMENTED** [review]
Thanks @jburel. Main concern is whether we want to make explicit changes to binder hub or keep updating the redirects to minimize the changes at the level of this repository.

A couple of additional thoughts related to the migration:

- in addition to the static IDR website, the top-level menu is (unfortunately) completely duplicated in https://github.com/IDR/deployment/blob/IDR-0.9.2/ansible/group_vars/omero-hosts.yml#L467 so the two locations will need to be sorted in parallel
- some content is also in https://github.com/IDR/idr-analysis.openmicroscopy.org/. Assuming this repository will be archived, is there any content that should be ported back here


Beyond https://idr.openmicroscopy.org/jupyter and https://idr-analysis.openmicroscopy.org/, are there other URLs in our domain we will need to maintain and redirect appropriately?

File: _includes/head-idr.html (hunk 1 above)
Comment (jburel) [reply _includes/head-idr.html:47]: We could.
If EBI writes some documentation about its deployment, we may, at stage, add a new entry,e.g. start the VAE and keep the <li><a href="/jupyter">Virtual Analysis Environment (VAE)</a></li> to deployment of EBI doc.

**Comment by jburel** [comment]
https://github.com/IDR/idr-analysis.openmicroscopy.org/: this doc will hopefully be replaced by the EBI doc when available.
* deployment page: no longer valid
* authentication phase: not really working.

**Comment by jburel** [comment]
Keeping ``/jupyter`` will also mean that we do not need to update https://github.com/IDR/deployment/blob/IDR-0.9.2/ansible/group_vars/omero-hosts.yml#L467

**Comment by sbesson** [comment]
@jburel should this be included in `prod93` or does it need more time/review?

**Comment by jburel** [comment]
I have kept ``/jupyter`` so if `` https://github.com/IDR/deployment/blob/IDR-0.9.2/ansible/group_vars/omero-hosts.yml#L467`` is not modified then it will still point to ``idr-analysis``
I think we can merge it as it is for this round and adjust again if required for next release

**Pull request merged**